### PR TITLE
ENG-3898 add list of component names to log

### PR DESCRIPTION
**Pull request by alepintus**

### CODE REVIEW

**Review by zonia3000 — APPROVED** [review]

File: src/commands/base-build.ts [hunk 1]
```diff
@@ -23,7 +23,22 @@ export abstract class BaseBuildCommand extends BaseExecutionCommand {
...
+
+    switch (componentType) {
+      case ComponentType.MICROSERVICE:
+        this.log(color.bold.blue(`Building ${componentsNames} microservices`))
+        break
+      case ComponentType.MICROFRONTEND:
+        this.log(color.bold.blue(`Building ${componentsNames} micro frontends`))
+        break
+      default:
+        this.log(color.bold.blue(`Building ${componentsNames} components`))
```
Comment (cecchisandrone) [inline src/commands/base-build.ts]: add ...

File: src/commands/base-build.ts [hunk 2]
```diff
@@ -23,7 +23,22 @@ export abstract class BaseBuildCommand extends BaseExecutionCommand {
...
+    const components = componentService.getComponents(componentType)
+
+    const componentsNames = components.map(comp => comp.name).join(', ')
+
+    switch (componentType) {
+      case ComponentType.MICROSERVICE:
+        this.log(color.bold.blue(`Building ${componentsNames} microservices`))
+        break
+      case ComponentType.MICROFRONTEND:
+        this.log(color.bold.blue(`Building ${componentsNames} micro frontends`))
```
Comment (cecchisandrone) [inline src/commands/base-build.ts]: add ...

File: src/commands/base-build.ts [hunk 3]
```diff
@@ -23,7 +23,22 @@ export abstract class BaseBuildCommand extends BaseExecutionCommand {
...
   ): Promise<void> {
-    this.log(color.bold.blue('Building components...'))
+    const componentService = new ComponentService()
+    const components = componentService.getComponents(componentType)
+
+    const componentsNames = components.map(comp => comp.name).join(', ')
+
+    switch (componentType) {
+      case ComponentType.MICROSERVICE:
+        this.log(color.bold.blue(`Building ${componentsNames} microservices`))
```
Comment (cecchisandrone) [inline src/commands/base-build.ts]: add ...

File: src/commands/base-build.ts (hunk 1 above)
Comment (alepintus) [reply src/commands/base-build.ts]: done

File: src/commands/base-build.ts (hunk 2 above)
Comment (alepintus) [reply src/commands/base-build.ts]: done

File: src/commands/base-build.ts (hunk 3 above)
Comment (alepintus) [reply src/commands/base-build.ts]: done

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**